src/App.tsx

```diff
@@ -11,7 +11,7 @@ function App() {
   const isBelowBreakpoint = useMediaQuery(1023);
   return (
     <div className="App">
-      <header>
+      <header aria-hidden="true">
```

Comment:
since the header only contains conditionally rendered background images, it makes sense to hide this from screen readers

Reply:
For this issue, you have added aria-hidden="true" to the header element, which hides the entire header from screen readers. To solve the issue of this decorative image receiving screen reader focus, remove aria-hiddn="true" and remove the value from the 'alt' attribute so that it contains an empty string. This causes screen readers to skip the image as it is marked as decorative.

Reply:
made the edits as suggested